library/entity/multipart/build.gradle

```
@@ -0,0 +1,25 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Multipart Entity API"
+	moduleName = "multipart"
+	id = "quilt_multipart"
```

```suggestion
	id = "quilt_entity_multipart"
```

This is entity related, it should state it.
There could be other stuff that could use the multipart wording like blocks (beacon with their base could be called multipart).

This also must be reflected in the package (`org.quiltmc.qsl.entity.multipart`).